<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lightning</string>
  </property>
  <property name="windowIcon">
   <iconset theme="lightningmc">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="miscHLayout"/>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="filesVLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionTarget_Folder"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddFiles"/>
   <addaction name="actionRemove_File"/>
   <addaction name="actionClear_Files"/>
   <addaction name="actionConvert_Files"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAddFiles">
   <property name="icon">
    <iconset theme="add">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>AddFiles</string>
   </property>
   <property name="toolTip">
    <string>Add Files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionRemove_File">
   <property name="icon">
    <iconset theme="remove">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Remove File</string>
   </property>
   <property name="toolTip">
    <string>Remove Selected File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionClear_Files">
   <property name="icon">
    <iconset theme="edit-clear">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Clear Files</string>
   </property>
   <property name="toolTip">
    <string>Clear File List</string>
   </property>
   <property name="shortcut">
    <string>Alt+Shift+C</string>
   </property>
  </action>
  <action name="actionConvert_Files">
   <property name="icon">
    <iconset theme="view-refresh">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Convert Files</string>
   </property>
   <property name="toolTip">
    <string>Convert Files</string>
   </property>
   <property name="shortcut">
    <string>Alt+Shift+C</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionCancel_Convertion">
   <property name="icon">
    <iconset theme="cancel">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Cancel Convertion</string>
   </property>
   <property name="toolTip">
    <string>Cancel Convertion</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionTarget_Folder">
   <property name="text">
    <string>Target Folder</string>
   </property>
   <property name="shortcut">
    <string>Alt+Shift+T</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
